config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -168,7 +168,7 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: patrick.ohly@intel.com
   - name: pull-node-dra
-    test_group_name: pull-kubernetes-node-e2e-crio-dra
+    test_group_name: pull-kubernetes-node-e2e-crio-cgrpv2-dra
     alert_options:
       alert_mail_to_addresses: eduard.bartosh@intel.com
   - name: pull-node-dra-containerd-1-7
```

Comment:
Would it make sense to do the same for `containerd` jobs in a separate PR?

Reply:
Yes! I hope to get some help on the containerd side.

Reply:
So containerd jobs for dra use the following config:

```
images:
  ubuntu:
    image_family: pipeline-1-29
    project: ubuntu-os-gke-cloud
    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/dra/init-containerd-1.7.yaml"
  cos:
    image_family: cos-stable
    project: cos-cloud
    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/dra/init-containerd-1.7.yaml"
```

This means that we run the test twice with two different images. I booted up a cos-stable cluster and I confirmed that this is actually cgroupv2.

I think the ubuntu one may be cgroup v1 as you usually need a special regexp to pick out a cgroupv2 image at the moment. (https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/containerd/image-config-serial.yaml)

I think we have coverage on the cgroupv2 side for cos but for ubuntu we are testing cgroupv1. If one wants to fix this they could add a new image for ubuntu to test cgroupv2 but not sure if its really necessary for DRA which is still in alpha.